docs/recipes.md

```diff
@@ -16,16 +16,19 @@ module `node:child_process`.
 #### `exec(command, callback)`
 
 When using `child_process.exec` without the `options` argument, use
-`shescape.quote` to escape all user input in the command string.
+`Shescape#quote` to escape all user input in the command string.
 
 ```javascript
 import { exec } from "node:child_process";
-import * as shescape from "shescape";
+import { Shescape } from "shescape";
 
-/* 1. Collect user input */
+/* 1. Set up */
+const shescape = new Shescape();
```

Comment:
Update the examples in these recipes to set `interpolation` and `flagProtection` to `false` if they're not required for the example(?)

Reply:
Resolved with e14f8e1421330600abcfa46dd5ffa9c2fe9e9fa4